run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+###############################################################################
+# Build docker full image and run ROS code interactively for debugging        #
+###############################################################################
+
+# Build docker image only up to base stage
+DOCKER_BUILDKIT=1 docker build \
+-t imu_humble \
+-f Dockerfile --target build .
+
+# Run docker image without volumes
+docker run -it --rm --net host \
+-v /dev/shm:/dev/shm \
+imu_humble
```

Comment:
I think this is very minor but, potentially, `run.sh` can accept an arg that specifies `dev` or `runtime` docker run.

Also, do we want to build the docker image every time we run the container? I think we can save time if we do not, specially if the amount of docker containers that defined our AV stack increases.

Let me know your comments on this one.

Reply:
`runtime.sh` and `dev.sh` docker convenience scripts have been implemented.